adoc/headers/queue.h

```diff
@@ -53,22 +53,25 @@ class queue {
 
   bool is_in_order() const;
 
-  template <typename Param> typename Param::return_type get_info() const;
+  template <typename Param>
```

Comment:
It's not important, but AFAIK those files were formatted via clang format.
In the future, we should commit a clang-format-config on we a style we like and stick with it.